src/execution_impl.rs

```diff
@@ -34,7 +34,10 @@ fn create_instance(limit: u64, module: &[u8], env: &Env) -> Result<Instance> {
 
     let base = BaseTunables::for_target(&Target::default());
     let tunables = LimitingTunables::new(base, Pages(max_number_of_pages()));
-    let store = Store::new_with_tunables(&Universal::new(compiler_config).engine(), tunables);
+    let engine = Universal::new(compiler_config)
+        .features(Features::new().threads(false)) // disable threads for determinism
```

Comment:
Note that per the [docs](https://docs.rs/wasmer/latest/wasmer/struct.Features.html#structfield.threads), this is `false` by default, but looking at the other features it seems it will be switched to `true` by default once it had been standardized, so I guess this is more future-proof.

Reply:
exactly :)